examples/typescript/src/Home.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const sleep = ms => new Promise(resolve => setTimeout(resolve, ms));
+
+class Home extends React.Component<{prefetch: (path: string) => void; stuff?: string; }> {
```

Comment:
We need a generic type for after component props:

```ts
import { AfterComponentProps } from 'after'
class Home extends React.Component< AfterComponentProps<{stuff?: string }> > {
```

Reply:
Couple of ways to do this. We could do something like: 

```tsx
import { RouteComponentProps } from 'react-router-dom'

export type AfterProps<Params> = RouteComponentProps<Params> & {
 prefetch: (pathname: string) => void;
 refetch: (ctx: any) => void;
}
```

Reply:
Are you open to maintaining types within this repo? Much preferred to having them at `@types`

Reply:
Hell yes.